src/Shell/QueueShell.php

```diff
@@ -65,6 +67,23 @@ public function initialize() {
 			}
 		}
 
+		//configure the logger if it is set to true
+		if (Configure::read('Queue.log')) {
+			Log::drop('debug');
+			Log::drop('error');
```

Comment:
`Once a configuration is created you cannot change it. Instead you should drop the configuration and re-create it using Cake\Log\Log::drop() and Cake\Log\Log::config().`

otherwise it logs both on queue.log and debug.log Any suggestions to separate the logic without dropping? bake also does that btw.